rest/model/createhost.go

```diff
@@ -40,6 +42,7 @@ func (createHost *CreateHost) BuildFromService(h interface{}) error {
 	case *host.Host:
 		createHost.DNSName = utility.ToStringPtr(v.Host)
 		createHost.IP = utility.ToStringPtr(v.IP)
+		createHost.IPv4 = utility.ToStringPtr(v.IPv4)
```

Comment:
v.IPv4 can potentially be "" (there is no omitempty). In that case, do we want to set it to "" instead of leaving it as a nil pointer? I see that that's what's being done for the other ip address, but I don't know if that was intentional. @ablack12 do you know if that is the convention?

Reply:
ToStringPointer doesn't set it to a nil pointer, it sets it to a pointer of the "" string. This is convention since we do this for every string.

Reply:
Right, I meant to ask if we should be leaving it as 'nil' instead of a pointer to "", so that if it is not set, the user sees nothing instead of seeing ` IPv4:    `.  But you're right, this seems to be the convention.